org.eclipse.winery.repository.ui/src/app/instance/instanceHeader/instanceHeader.component.html

```diff
@@ -14,12 +14,10 @@
 <div class="top" [class.twolines]="needTwoLines">
     <img *ngIf="imageUrl" src="{{ imageUrl }}" class="nodeTypeImageIcon">
     <div class="informationContainer">
-        <div id="component_name" class="name editable editable-click">
-            {{ toscaComponent.localName }}
-        </div>
-        <div id="component_namespace" class="namespace editable editable-click">
-            {{ toscaComponent.namespace }}
-        </div>
+        <winery-property-rename [toscaComponent]="toscaComponent" [currentValue]="toscaComponent.localName" [id]="'id'"
+                                [propertyName]="'id'"></winery-property-rename>
```

Comment:
You don't need to separately pass the `toscaComponent` and the `currentValue`. The `toscaComponent` already includes this because you pass it the value of `toscaComponent.localName`

`[id]="'id'"` => `[id]="'local name'"`

Reply:
1. done
2. The id is the name of the property in the sectionsData i get from the backend and that is 'id'